UNITTESTS/fakes/drivers/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+# Copyright (c) 2021 ARM Limited. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+add_subdirectory(PwmOut)
```

Comment:
The macro is available here but there is no point in using it if we're lisitng the direcotries by name, it's faster to just call add_subdirectory directly. The macro would only be useful if we allowed globbing of directory names.